frontend/public/components/factory/list-page.jsx

```diff
@@ -242,22 +246,24 @@ export const FireMan_ = connect(null, { filterList })(
       return (
         <>
           {/* Badge rendered from PageHeading only when title is present */}
-          <PageHeading
-            title={title}
-            badge={title ? badge : null}
-            className={classNames({ 'co-m-nav-title--row': createLink })}
-          >
-            {createLink && (
-              <div
-                className={classNames('co-m-pane__createLink', {
-                  'co-m-pane__createLink--no-title': !title,
-                })}
-              >
-                {createLink}
-              </div>
-            )}
-            {!title && badge && <div>{badge}</div>}
-          </PageHeading>
+          {showTitle && (
```

Comment:
showTitle is used a lot of places and if any of those uses expect the create link, they will no longer get it.  I might have to add another flag to hide the title and create Link.

Reply:
Removed showTitle here and created new flag hidePageHeading